app/controllers/application_controller.rb

```diff
@@ -76,12 +76,13 @@ def signed_in?
 
   def authenticate_user!
     if not_signed_in?
-      flash[:error] = 'You need to sign in for access to this page.'
+      flash[:error] = "You need to sign in for access to this page."
       return_to_param = make_return_to("Previous", request.fullpath)
       session[:return_to] = return_to_param
       redirect_to root_url(return_to: return_to_param)
-    elsif session[:return_to]
```

Comment:
@pkarman I removed this because I wasn't sure why it was here - thoughts? Happy to re-introduce and write a test for it if we need it :)